tgui/packages/tgui/components/Button.js

```diff
@@ -21,6 +21,8 @@ export const Button = props => {
     icon,
     iconRotation,
     iconSpin,
+    iconColor,
+    iconRight,
```

Comment:
Replace this property with `iconPosition="right"`, because `iconRight` reads as if you want to put an icon into this prop, and not a boolean. It's also consistent with `tooltipPosition`.

You only need to handle one case, which is `"right"`, so code should remain basically the same, `=== 'right'`, and `!== 'right'`.

Reply:
Done, hopefully right.